src/content/reference/react-dom/client/index.md

```diff
@@ -1,22 +1,23 @@
 ---
-title: Client React DOM APIs
+title: API Klien React DOM
 ---
 
 <Intro>
 
-The `react-dom/client` APIs let you render React components on the client (in the browser). These APIs are typically used at the top level of your app to initialize your React tree. A [framework](/learn/start-a-new-react-project#production-grade-react-frameworks) may call them for you. Most of your components don't need to import or use them.
+
+API `react-dom/client` memungkinkan Anda me-*render* komponen React di sisi klien (di peramban). Biasanya API ini digunakan pada level atas aplikasi React untuk menginisialisasi pohon React. Sebuah [*framework*](/learn/start-a-new-react-project#production-grade-react-frameworks) mungkin memanggilnya untuk Anda. Biasanya komponen Anda tidak perlu mengimpor atau menggunakannya.
 
 </Intro>
 
 ---
 
-## Client APIs {/*client-apis*/}
+## API Klien {/*client-apis*/}
 
-* [`createRoot`](/reference/react-dom/client/createRoot) lets you create a root to display React components inside a browser DOM node.
-* [`hydrateRoot`](/reference/react-dom/client/hydrateRoot) lets you display React components inside a browser DOM node whose HTML content was previously generated by [`react-dom/server`.](/reference/react-dom/server)
+* [`createRoot`](/reference/react-dom/client/createRoot) memungkinkan untuk membuat akar *(root)* yang digunakan untuk menampilkan komponen React di dalam simpul *(node)* DOM peramban.
+* [`hydrateRoot`](/reference/react-dom/client/hydrateRoot) memungkinkan untuk menampilkan komponen React di dalam simpul DOM peramban  yang telah dihasilkan oleh [`react-dom/server`.](/reference/react-dom/server)
```

Comment:
```suggestion
* [`createRoot`](/reference/react-dom/client/createRoot) memungkinkan untuk membuat akar (*root*) yang digunakan untuk menampilkan komponen React di dalam simpul (*node*) DOM peramban.
* [`hydrateRoot`](/reference/react-dom/client/hydrateRoot) memungkinkan untuk menampilkan komponen React di dalam simpul DOM peramban  yang telah dihasilkan oleh [`react-dom/server`.](/reference/react-dom/server)
```

As far as I remember, the brackets `()` shouldn't be in *italic*, CMIIW @mhaidarhanif.

Reply:
Yes, also missed this part.